test/libsolidity/util/TestFileParser.h

```diff
@@ -120,8 +120,6 @@ class TestFileParser
 		std::string m_line;
 		std::string::const_iterator m_char;
 
-		std::string m_currentLiteral;
```

Comment:
This was unused, because of the above `selectToken` behaviour the token/literal fields are merged into the token field. We could improve this to work nicely, but this change is the simplest to get it going.